spec/lib/vanagon/platform_spec.rb

```diff
@@ -24,10 +24,10 @@
         :os_name                        => "el",
         :os_version                     => "5",
         :architecture                   => "i386",
-        :output_dir                     => "el/5/products/i386",
+        :output_dir                     => "el-5-i386",
         :output_dir_with_target         => "el/5/thing/i386",
         :output_dir_empty_string        => "el/5/i386",
-        :source_output_dir              => "el/5/products/SRPMS",
+        :source_output_dir              => "el-5-srpms",
         :source_output_dir_with_target  => "el/5/thing/SRPMS",
         :source_output_dir_empty_string => "el/5/SRPMS",
```

Comment:
@underscorgan  e already have quite a few tests surrounding old and new behavior, luckily!